### DISCUSSION TITLE
[Projects beta] Search by issue title

### DISCUSSION THREAD

**Discussion by marisamorby** [community/community#18108] [Projects and Issues] [answered]
We have a shared project board across our design and research team. We often need to search for issues with keywords from the title, but this isn't currently possible.

**Comment by vvdavidova** [comment, marked as answer]
The way I am solving this is the following: 
Created a view, where I have absolutely no filters. Pressing Ctrl + F triggers search. Type in the keyword you want and voila. 

I know, not ideal, as it means having one more view, where 'nothing' is really happening, but in my team, currently we do keep it, so we can search.

**Reply by marisamorby** [reply]
Oh that works pretty well actually, thank you!

**Reply by mgstauffer** [reply]
Seems this still isn't possible?! On the projects page that shows list of projects for my repo, there's a search box at the top which is prepopulated with "is:open" , but it doesn't find anything even for words that I know are in issues.

@vvdavidova As for your fix, could you help me out? What does it mean to create a view?
